docs/docs/references/configuration/cli/trivy.md

```diff
@@ -43,6 +43,7 @@ trivy [global flags] command [flags] target
 
 ### SEE ALSO
 
+* [trivy clean](trivy_clean.md)	 - Remove cached files
```

Comment:
Add new page to `mkdocs.yml`, please

Reply:
Added https://github.com/aquasecurity/trivy/pull/6993/commits/b50895b9b5162b8372972338a5c6238e5e3f06f5